docker-compose.yml

```diff
@@ -3,17 +3,17 @@ version: '2.3'
 # versions: change these in order to pin versions for a certain releases
 x-versions:
   services: &IMAGE_RAIDEN_SERVICES_VERSION
-    image: raidennetwork/raiden-services:v0.14.0
+    image: raidennetwork/raiden-services:v0.14.1
```

Comment:
The services release is not there yet

Reply:
Now it is: https://hub.docker.com/layers/139191165/raidennetwork/raiden-services/v0.14.1/images/sha256-1be60dcc2979eef911d2ad927c57c57bcdfd93ba1bd2c1ce92b4108b000c465d?context=explore